lte/gateway/c/core/oai/tasks/amf/amf_app_ue_context.cpp

```diff
@@ -139,6 +139,7 @@ ue_m5gmm_context_s* amf_create_new_ue_context(void) {
     OAILOG_ERROR(LOG_AMF_APP, "Failed to allocate memory for UE context \n");
     return NULL;
   }
+  memset(new_p, 0, sizeof(ue_m5gmm_context_s));
```

Comment:
add unit test coverage for `amf_create_new_ue_context`

Reply:
there was no issue in  ue_m5gmm_context_s initialization, safer side I added.
ue_m5gmm_context_s  struct   allocated with c++ constructor(new ue_m5gmm_context_s()  ),properly all members are initialized.validated with UT.